csl-data.json

```diff
@@ -1,431 +1,452 @@
-                                        "type": [
-                                            "string",
-                                            "number" 
-                                        ] 
-                                    },
-                                    "maxItems": 3 
-                                },
-                                "maxItems": 2 
+                            "literal": {
+                                "type": "string"
+                            }
+                        },
+                        "required": [
+                            "literal"
+                        ]
+                    },
+                    {
+                        "type": "object",
+                        "properties": {
+                            "family": {
+                                "type": "string"
                             },
-                            "season": {
+                            "given": {
+                                "type": "string"
+                            },
+                            "dropping-particle": {
+                                "type": "string"
+                            },
+                            "non-dropping-particle": {
+                                "type": "string"
+                            },
+                            "suffix": {
+                                "type": "string"
+                            },
+                            "comma-suffix": {
                                 "type": [
                                     "string",
-                                    "number" 
-                                ] 
+                                    "number",
+                                    "boolean"
+                                ]
                             },
-                            "circa": {
+                            "static-ordering": {
                                 "type": [
                                     "string",
                                     "number",
-                                    "boolean" 
-                                ] 
-                            },
-                            "literal": {
-                                "type": "string" 
+                                    "boolean"
+                                ]
                             },
-                            "raw": {
-                                "type": "string" 
+                            "parse-names": {
+                                "type": [
+                                    "string",
+                                    "number",
+                                    "boolean"
+                                ]
                             }
                         },
-                        "additionalProperties" : false 
+                        "required": [
+                            "family"
+                        ]
```

Comment:
I'm not sure we can require "family".

I just checked and a Zotero user can store just a given name, which gives this as exported CSL JSON:

```
			{
				"family": "Nielsen",
				"given": "Jens"
			},
			{
				"family": "",
				"given": "bla"
			}
```

Other implementations might leave out the field entirely.